packages/react/package.json

```diff
@@ -12,7 +12,9 @@
   "exports": {
     ".": "./dist/index.js"
   },
-  "types": "src/index.ts",
+  "files": [
+    "dist"
```

Comment:
I noticed CHANGELOG was previously included via npmignore but not here, curious if you think it's worth including? cc @alvrs

Reply:
no strong opinion here

Reply:
CHANGELOG, package.json, README.md and LICENSE.md are automatically included, regardless of whether they are explicitly listed in files.

Reply:
going to go with no changelog for now to keep package smaller, can revisit in the future